en/docs/guides/extracting-data-from-static-sources-in-real-time.md

```
@@ -107,32 +107,35 @@ Let's try out the example where you want to view the online bookings saved in a
+          PRIMARY KEY (ref));
+        ```
           
     4. [Open VSCode with WSO2 Integrator: SI extension installed](../develop/si-for-vscode-overview.md).
     
-    5. Download the `cdc-mysql`Siddhi extension for WSO2 Integrator: SI VSCode extension. For instructions, see [Installing Siddhi Extensions](../../develop/installing-siddhi-extensions/#installing-an-extension).
+    5. Download the `cdc-mysql` Siddhi extension for WSO2 Integrator: SI VSCode extension. For instructions, see [Installing Siddhi Extensions](../develop/installing-siddhi-extensions.md#installing-an-extension).
     
-    6. In WSO2 Integrator: SI Tooling, open a new file. Copy and paste the following Siddhi application to it.
+    6. In the VSCode editor (with the WSO2 Integrator: SI extension installed), open a new file. Copy and paste the following Siddhi application to it.
     
         ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)